### PR TITLE
support reading the line content and audio cues in diff review mode

### LINKED ISSUE #171200: support screen reader reading the line and audio cues when go to next/previous diff are used in diff view
Fulfilled for normal editors in these:
https://github.com/microsoft/vscode/issues/166518
https://github.com/microsoft/vscode/issues/164988

### PR DESCRIPTION
**Pull request by meganrogge**
fixes https://github.com/microsoft/vscode/issues/171200

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts [hunk 1]
```diff
@@ -79,6 +79,8 @@ class MockQuickInputService implements Partial<IQuickInputService> {
 	}
 }
 
+
+
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts:83]:
```suggestion
```

File: src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts [hunk 2]
```diff
@@ -87,6 +89,8 @@ class TestTerminalExtensionService extends TestExtensionService {
 	readonly _onDidChangeExtensions = new Emitter<void>();
 }
 
+
+
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts:93]:
```suggestion
```

File: src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts (hunk 1 above)
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/test/browser/terminalProfileService.integrationTest.ts:83]:
```suggestion
```

**Comment by isidorn** [comment]
Audio cues make perfect sense in diff review mode and I like that change.
But reading line content - we should align this with the current F7 experience in the diff editor.

Also assigning @hediet to review since Alex is on vacation.

**Review by hediet — DISMISSED** [review]

**Review by isidorn — DISMISSED** [review]

**Review by andreamah — APPROVED** [review]

**Pull request merged**